stetho/src/main/java/com/facebook/stetho/inspector/elements/android/FragmentDescriptor.java

```diff
@@ -64,6 +68,8 @@ protected int onGetChildCount(Object element) {
 
   @Override
   protected Object onGetChildAt(Object element, int index) {
+    verifyAccess();
```

Comment:
Summary of outside-of-github conversation: We're going to call verifyAccess() in DOMProvider.getDescriptor(), but not in the NodeDescriptor method calls. This will remove the TLS access cost and the verifyAccess() repetition. NodeDescriptors should then call verifyAccess() in 'dangerous' functions, such as anything that *writes* to them (e.g. setAttributesAsText), particularly if they're low-volume. NodeDescriptor will still implement ThreadBound.